### DISCUSSION TITLE
How to use template types in the Patterns API

### DISCUSSION THREAD

**Discussion by annezazu** [WordPress/developer-blog-content#58] [Topic Ideas]
Basically create a how to/application style post around using this new functionality: https://github.com/WordPress/gutenberg/pull/45814

**Comment by justintadlock** [comment]
I'm very interested in taking this tutorial on.

Currently, there are no UI pieces to this, and this is one of those features that requires UI to be useful or to even really explain what it does.  This will go hand-in-hand with an updated "add template" flow: https://github.com/WordPress/gutenberg/issues/42325, which is still under development.

**Reply by bph** [reply]
Go for it @justintadlock  👏
- [x] Convert it to an issue
- [x] Assign it to yourself
- [x] Add it to the Project Board under 'needs approval'

**Comment by annezazu** [comment]
If this gets addressed it would be interesting to include too: https://github.com/WordPress/gutenberg/issues/47208

**Comment by justintadlock** [comment]
PR for adding patterns to the template start modal: https://github.com/WordPress/gutenberg/pull/47322

**Comment by bph** [comment]
This discussion is now locked. For further discussion, please comment on the issue
[Tutorial on using template types in the Patterns API#65](https://github.com/WordPress/developer-blog-content/issues/65)
